src/Input/Keybindings.re

```diff
@@ -111,22 +158,117 @@ module Legacy = {
     };
 };
 
-let of_yojson_with_errors = json => {
-  switch (json) {
-  // Current format:
-  // [ ...bindings ]
-  | `List(bindingsJson) =>
-    let res = bindingsJson |> Internal.of_yojson_with_errors;
-    Ok(res);
-  // Legacy format:
-  // { bindings: [ ..bindings. ] }
-  | `Assoc([("bindings", `List(bindingsJson))]) =>
-    let res = bindingsJson |> Internal.of_yojson_with_errors;
-
-    Ok(res)
-    |> Stdlib.Result.map(((bindings, errors)) => {
-         (Legacy.upgrade(bindings), errors)
-       });
-  | _ => Error("Unable to parse keybindings - not a JSON array.")
+let keyToSdl =
+  EditorInput.Key.(
+    {
+      fun
+      | Escape => "Escape"
+      | Return => "Return"
+      | Up => "Up"
+      | Down => "Down"
```

Comment:
`Sdl` is really not a thing I expect to be able to convert a key to. Perhaps it would make sense to further qualify it as `SdlName`?